keyboards/zeal60/zeal60_api.h

```diff
@@ -43,6 +43,7 @@ enum zeal60_command_id
 
 enum zeal60_keyboard_value_id
 {
-	id_uptime = 0x01
+  id_uptime = 0x01,
+  id_firmware_version
```

Comment:
Is @Wilba6582 okay with this change?

Reply:
He is, we have talked about it.

Reply:
Well, I'd prefer to have @Wilba6582 confirm this himself, to be absolutely certain.

Reply:
👍 I'll go bother him :P

Reply:
@drashna if you really need me to approve stuff, make me an official reviewer. Otherwise you can assume I'm watching if mentioned.

Reply:
Well, for this, since it affects your code, I'd rather have explicit approval, rather than assuming. 
(Because I've already made that mistake, once, and I don't like to repeat mistakes, when possible)

Also, anyone can "approve" a PR, so that may work in the meanwhile.  
As in, if it affects your code, or is VIA related, and we see that approval check/tick mark, we'll merge it. If not, we'll make sure you're tagged in the PR and wait for your approval.    

But I think this may merit some more discussion on discord or the like. 

Otherwise, I know that there has been talk about get makers and others more actively involved in the development process.